.buildkite/release-package-registry-distribution.yml

```diff
@@ -45,3 +45,17 @@ steps:
     depends_on:
       - step: "validate"
         allow_failure: false
+
+  - trigger: "package-storage-infra-update-package-registry"
+    label: ":esbuild: Downstream - Update package registry"
+    key: "downstream-update-package-registry"
+    async: true
+    build:
+      branch: "main"
+      env:
+        DRY_RUN: "$DRY_RUN"
+        DRAFT_PR: "false"
+        EPR_VERSION: ""
+    depends_on:
+      - step: "release-distribution"
+        allow_failure: false
```

Comment:
This pipeline should not have this new step.

This step should be added into the main pipeline `.buildkite/pipeline.yml`
It needs to replace this step:
https://github.com/elastic/package-registry/blob/e074d1811cc5a4e72b7152b802897ef87474e287/.buildkite/pipeline.yml#L77-L91

Also adding the `if` to just run this pipeline when a tag is pushed

To test this, you could trigger a commit with this step ensuring that `DRY_RUN=true`and removing `if` temporarily to trigger this pipeline in a PR build.


As part of this PR, the next files (or folders) could be removed too:
- .buildkite/scripts/trigger-jenkins-job.sh
- .buildkite/scripts/triggerJenkinsJob

Reply:
done